assets/panel/debugger.properties

```diff
@@ -24,6 +24,11 @@ copySource.accesskey=y
 copySourceUri2=Copy source URI
 copySourceUri2.accesskey=u
 
+# LOCALIZATION NOTE (copySourceText): This is the text that appears in the
+# context menu to copy the source text of file open.
+copySourceText=Copy source code
```

Comment:
Please use `copySourceText.label` for this string ID (also in the comment above). That's the best way to name strings, unfortunately a lot of the old ones use "foo" and "foo.accesskey".